camel/models/litellm_model.py

```diff
@@ -0,0 +1,98 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from typing import Any, Dict, List, Optional, Union
+
+from litellm import completion
+from litellm.utils import (
+    CustomStreamWrapper,
+    ModelResponse,
+)
```

Comment:
move the import here into __init__ since litellm would be an optional platform

Reply:
> move the import here into **init** since litellm would be an optional platform

This file is specifically written for the LiteLLM platform. Is it necessary to move the import into the init in this context? Doing so might introduce unnecessary complexity to the code structure.